### DISCUSSION TITLE
Discord RPC?

### DISCUSSION THREAD

**Discussion by ghost** [kraxarn/spotify-qt#244] [Ideas]
It's not something practical or necessary but it would be nice to have Discord RPC as like a QOL feature in the future

**Comment by kraxarn** [comment]
As i rarely use Discord myself, and never the official client, I don't see me implementing this feature any time soon, but if someone else wants to, I'm open to it.

**Comment by beedell-roke** [comment]
> https://github.com/kraxarn/spotify-qt/issues/230#issue-1908087477

Note that https://github.com/OpenAsar/arrpc now exists, so integration shouldn't be hideously difficult anymore.
